README.md

```diff
@@ -1,48 +1,36 @@
-<div align="center">
-    <img alt="JetKVM logo" src="https://jetkvm.com/logo-blue.png" height="28">
+JetKVM
+<div align="center"> <img alt="JetKVM logo" src="https://jetkvm.com/logo-blue.png" height="28"> </div>
 
-### KVM
+Links
+Discord | Website | Issues | Docs
```

Comment:
Shouldn't still this be
```markdown
[Discord](https://jetkvm.com/discord) | [Website](https://jetkvm.com) | [Issues](https://github.com/jetkvm/cloud-api/issues) | [Docs](https://jetkvm.com/docs)
```

Looks like you did a copy-paste from the rendered screen and lost all the markdown.